packages/cadl-ranch-specs/http/type/model/visibility/main.tsp

```diff
@@ -30,6 +32,206 @@ model VisibilityModel {
   deleteProp: boolean;
 }
 
+@doc("Base model")
+model BaseModel {}
+
+@doc("Derived model")
+model DerivedModel extends BaseModel {
+  @doc("Required property")
+  resourceName: string;
+}
+
+@doc("Collection item model")
+model CollectionItem {
+  @doc("Required resource id")
+  requiredId: int32;
+}
+
+@doc("Fixed string enum")
+enum FixedStringEnum {
+  One: "1",
+  Two: "2",
+  Four: "4",
+}
+
+@doc("Extensible enum")
+union ExtensibleEnum {
+  string,
+  One: "1",
+  Two: "2",
+  Four: "4",
+}
+
+@doc("Output model with readonly properties.")
```

Comment:
As I said, don't do copy-paste. Use the doc here to let others know what you are expecting to test.

Reply:
fixed

Reply:
For example,

Test `@visibility("readonly")` on different types of property. All are not expected in the payload when the model is input, while expected in the payload when the model is output.

Reply:
And `@doc` is deprecated, use `/**   */` pattern instead. You could see many examples in other specs.